client/state/selectors/is-directly-uninitialized.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * Tells whether the Directly RTM widget has started initialization
+ *
+ * @see lib/directly for more about Directly
+ *
+ * @param  {Object}  state  Global state tree
+ * @return {Boolean}        Whether the widget is waiting to be initialized
+ */
+
+/**
+ * Internal dependencies
+ */
+import { STATUS_UNINITIALIZED } from 'state/help/directly/constants';
+
+export default function isDirectlyUninitialized( state ) {
+	return state.help.directly.status === STATUS_UNINITIALIZED;
```

Comment:
Yeah, allowing re-tries when re-visiting /help/contact is a great idea. Since we're under the gun here on deadline, is it OK with you to push this to a "Future features" list if we stick with Directly longer-term? This would likely come up regardless if/when we refactor the `HelpContact` component.